ecs-agent/daemonimages/csidriver/driver/node.go

```diff
@@ -41,12 +243,15 @@ func newNodeService() nodeService {
 	}
 
 	return nodeService{
-		mounter: nodeMounter,
+		mounter:          nodeMounter,
+		deviceIdentifier: newNodeDeviceIdentifier(),
+		inFlight:         internal.NewInFlight(),
 	}
 }
 
 func (d *nodeService) NodeGetVolumeStats(ctx context.Context, req *csi.NodeGetVolumeStatsRequest) (*csi.NodeGetVolumeStatsResponse, error) {
-	klog.V(4).InfoS("NodeGetVolumeStats: called", "args", *req)
+	klog.InfoS("NodeGetVolumeStats: called")
```

Comment:
Are we getting rid of the args and req?

Reply:
(will add in a future pr with the updated build process)